src/ui/media/MediaLibrary.as

```diff
@@ -422,20 +422,26 @@ spriteFeaturesFilter.visible = false; // disable features filter for now
 			var item:MediaLibraryItem = resultsPane.getChildAt(i) as MediaLibraryItem;
 			if (item && item.isHighlighted()) {
 				var md5AndExt:String = item.dbObj.md5;
+				var obj:Object = null;
 				if (assetType == 'extension') {
 					whenDone(item.dbObj.extension);
 				} else if (md5AndExt.slice(-5) == '.json') {
 					io.fetchSprite(md5AndExt, whenDone);
 				} else if (assetType == 'sound') {
 					io.fetchSound(md5AndExt, item.dbObj.name, whenDone);
-				} else {
-					var obj:Object = {
+				} else if (assetType == 'costume') {
+					obj = {
 						centerX: item.dbObj.info[0],
 						centerY: item.dbObj.info[1],
 						bitmapResolution: 1
 					};
 					if (item.dbObj.info.length == 3)
 						obj.bitmapResolution = item.dbObj.info[2];
+
+					io.fetchImage(md5AndExt, item.dbObj.name, 0, whenDone, obj);
+				} else { // assetType == backdrop
+					if (item.dbObj.info.length == 2 && item.dbObj.info[0] == 960 && item.dbObj.info[1] == 720)
```

Comment:
Since we're so engrained with 480x360 and even high-resolution bitmaps have been abandoned I don't think that will be necessary (at least in this codebase).